app/src/theme/typography.scss

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
Btw, this shouldn't be necessary unless you want to change the default Canopy font (which I think is Noto sans). But if you want roboto that's cool.